Makefile

```diff
@@ -119,6 +148,22 @@ st: run-etcd dist/calicoctl docker calico_test/.calico_test.created busybox.tar
 	           calico/test \
 	           sh -c 'cp -ra tests/st/* /tests/st && cd / && nosetests $(ST_TO_RUN) -sv --nologcapture --with-timer $(ST_OPTIONS)'
 
+## Run the STs in a container using etcd with SSL certificate/key/CA verification.
+st-ssl: run-etcd-ssl dist/calicoctl docker calico_test/.calico_test.created busybox.tar calico-node.tar routereflector.tar
+	docker run --uts=host \
+	           --pid=host \
+	           --net=host \
+	           --privileged \
+	           -e HOST_CHECKOUT_DIR=$(HOST_CHECKOUT_DIR) \
+	           -e ETCD_SECURITY_TYPE=ssl \
+	           -e HOST_CERT_DIR=`pwd`/certs \
```

Comment:
This is a bit funny.  Both this HOST_CERT_DIR and the volume are required for non-DinD tests.

When using DinD, the Docker container hosts have `calico-docker` mounted as  `/code`. Passing the etcd certs to the `calicoctl node` command works fine because we know we can set the etcd environment variable paths from `/code/certs`.

However, when not using DinD, there are two things that happen:
1) Volume mounts during `docker run` will copy directories from the host since the st container uses the hosts docker socket.
2) The `calicoctl node` command is run from the st container, which checks the existence of the etcd certs on the machine that run the command (which is the st container).  This fails without mounting the hosts's certs directory as the same path in the test-container.

The `HOST_CERT_DIR` environment variable is needed to have a reference to the certs location on the host when mounting the cert files in calicoctl node.